src/Illuminate/Auth/Events/Lockout.php

```diff
@@ -6,21 +6,14 @@
 
 class Lockout
 {
-    /**
-     * The throttled request.
-     *
-     * @var \Illuminate\Http\Request
-     */
-    public $request;
-
     /**
      * Create a new event instance.
      *
      * @param  \Illuminate\Http\Request  $request
```

Comment:
Do we want to add the docblock here that we would otherwise lose?

Reply:
good catch, thanks